src/extractcode/extract.py

```diff
@@ -125,7 +125,7 @@ def extract(location, kinds=extractcode.default_kinds, recurse=False, replace_or
     """
     processed_events = []
     processed_events_append = processed_events.append
-    for event in extract_files(location, kinds, recurse):
+    for event in extract_files(location, kinds, recurse, ignored_extensions):
```

Comment:
is `ignored_extensions` always a file extention? What if I do something like `--ignore=specific-file.tar.gz`